File: config/src/main/java/org/wcl/mfn/config/url/UrlConfig.java
package org.wcl.mfn.config.url;

import org.springframework.beans.factory.annotation.Value;
import org.springframework.context.annotation.*;

@Configuration
@PropertySource("classpath:url/url.properties")
public class UrlConfig {
    @Value("${url.home}")
    private String homeUrl;

    @Value("${url.tools.contract-calculator}")
    private String contractCalculatorUrl;

    public String homeUrl() {
        return homeUrl;
    }

    public String contractCalculatorUrl() {
        return contractCalculatorUrl;
    }
}

File: config/src/main/kotlin/org/wcl/mfn/config/url/UrlConfig.kt
package org.wcl.mfn.config.url

import org.springframework.beans.factory.annotation.Value
import org.springframework.context.annotation.*

@Configuration
@PropertySource("classpath:url/url.properties")
open class UrlConfig {
    @Value("\${url.home}")
    private val homeUrl: String? = null

    @Value("\${url.tools.contract-calculator}")
    private val contractCalculatorUrl: String? = null

    fun homeUrl(): String? {
        return homeUrl
    }

    fun contractCalculatorUrl(): String? {
        return contractCalculatorUrl
    }
}
